apps/artemis-web/libs/next-csrf.js

```diff
@@ -2192,9 +2191,9 @@ var setup = function (handler, _a) {
         res = isApi
           ? args[1] // (req, *res*)
           : args[0].res;
+
         reqCsrfToken = tokens.create(csrfSecret);
         reqCsrfTokenSigned = cookieSignature.sign(reqCsrfToken, secret);
```

Comment:
we use `secret` and `csrfSecret` inconsistently

Reply:
This is not my code. This is a library I forked. csrfSecret and secret are different usually.. I set them to be the same in order to avoid an extra env var